### ISSUE TITLE
Local multiplayer in Mario Party Superstars broken over the last couple of weeks

### ISSUE THREAD

**Issue by kamakazedan** [yuzu-emu/yuzu#10298]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1433 / Early Access 3589

### Description of Issue

After joining a local room and then initialising the game (after everyone has joined) the host sees a black screen and clients see a loading wheel in the bottom right corner.

Using LAN connection between host & clients, swtiching who hosts makes no difference.

Other local play games are functioning without any issues e.g. Super Mario 3D World

It was working without any issues two weeks ago.

![01006fe013472000_2023-05-14_21-35-13-351](https://github.com/yuzu-emu/yuzu/assets/127953/2e261fee-3b5d-47a7-85b9-a9e964ed3f27)
![01006fe013472000_2023-05-14_21-15-56-250](https://github.com/yuzu-emu/yuzu/assets/127953/bdda9741-a624-4ba9-908e-b165b8763a64)

### Expected Behavior

When the host presses "Start with These Players", the game loads.

### Reproduction Steps

Join a multipler room
Have clients connect
Launch Mario Party Superstars
Host creates a local play room with no password
Clients join the local play room
Host presses "Start with These Players"
Host sees black screen
After about 10 seconds "Communication was interrupted" pops up

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11472899/yuzu_log.txt)

### System Configuration

CPU: Ryzen 7 5800X
GPU/Driver: RTX 3090 (531.68)
RAM: 32GB
OS: Windows 10

**Comment by FearlessTobi** [comment]
There have been absolutely no changes to multiplayer code lately. Please make sure everyone is using the same version and your connection is stable.

**Comment by kamakazedan** [comment]
Ok thanks for confirming, I've done more digging and it's the Steam Deck causing issues.
Strangely Super Mario 3D world was not impacted but Mario Kart 8 Deluxe is doing the same thing.
Even running via a USB-C hub with ethernet (I tried wifi & ethernet) the steam deck can no longer maintain a connection ingame although the yuzu lobby never drops.
Would this be anything related to the linux build or just some niche steam deck issue (pings on the steam deck seem solid)?

**Comment by FearlessTobi** [comment]
This has to be some weird steam deck edge case.
I honestly have no idea what could be causing this, especially when it worked before.

I'm going to close this for now, as there is nothing we can do on our side to fix this.
What cou can try is downloading an older build and testing with it, but it's unlikely that it will change anything.

**Comment by varialbe** [comment]
Hello, me and my friend are both using windows 10/11 and experience no issues at all in other games (regarding online play) but we face the exact same issue. We tried switching hosts and also using different versions of the game (while us always have the same version of course). We have also tried using the early access builds, which also resulted in us facing the exact same issue. We tried using yuzu EA 3910 & yuzu Mainline 1575. Reproduction steps are expected behavior are the same.
